crawler/http/crawler/settings.py

```diff
@@ -0,0 +1,33 @@
+# Scrapy settings for crawler project
+#
+# For simplicity, this file contains only settings considered important or
+# commonly used. You can find more settings consulting the documentation:
+#
+#     https://docs.scrapy.org/en/latest/topics/settings.html
+#     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
+#     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
+
+BOT_NAME = "crawler"
+
+SPIDER_MODULES = ["crawler.spiders"]
+NEWSPIDER_MODULE = "crawler.spiders"
+
+
+# Crawl responsibly by identifying yourself (and your website) on the user-agent
+USER_AGENT = "sycamore crawler (+https://github.com/aryn-ai/crawler)"
```

Comment:
Not a big deal, but might be better to point to sycamore repo, since the separate crawler repo presumably won't exist/won't be public.

Reply:
Done.